tests/pytorch/fpga/test_conv2d_fpga.py

```diff
@@ -0,0 +1,164 @@
+# Tests Naive convolutions for FPGA
+
+from dace.transformation.interstate import FPGATransformSDFG
+
+import torch
+import torch.nn as nn
+import argparse
+import numpy as np
+
+from daceml.pytorch import DaceModule, dace_module
+import pytest
+import dace
+from daceml.util import utils
+from dace.transformation.interstate import FPGATransformSDFG, InlineSDFG
+from daceml.transformation import InputToConstant
+from dace.transformation.dataflow import streaming_memory as sm
+from dace.transformation.dataflow import PruneConnectors
+from multiprocessing import Process, Queue
+
+import daceml.onnx as donnx
+
+donnx.default_implementation = "pure"
```

Comment:
Please don't overwrite the global default implementation like this.
We have the `@pytest.mark.pure` decorator for tests that will set the implementation to pure and reset it afterwards.